mmv1/products/vertexai/api.yaml

```diff
@@ -103,6 +103,238 @@ objects:
+      - !ruby/object:Api::Type::String
+        name: 'displayName'
+        required: true
+        description: |
+          The user-defined name of the Endpoint. The name can be up to 128 characters long and can be consist of any UTF-8 characters.
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: The description of the Endpoint.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'deployedModels'
+        input: true
+        description: |
+          The list of models deployed in this Endpoint
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'id'
+            description: The resource name of the deployed model.
+            output: true
+          - !ruby/object:Api::Type::String
+            name: 'model'
+            required: true
+            description: |
+              The name of the Model that this is the deployment of.  
+              Note that the Model may be in a different location than the DeployedModel's Endpoint.
+          - !ruby/object:Api::Type::String
+            name: 'createTime'
+            description: |
+              The timestamp of when the endpoint was created in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+            output: true
+            input: false
+          # TODO: Add explanationSpec and object(ExplanationSpec) here
+          # Reason for not including: Very deeply nested objects within objects which requires further thought to support readibility and deployment.
+          - !ruby/object:Api::Type::String
+            name: 'serviceAccount'
+            description: |
+              The service account that the DeployedModel's container runs as. Specify the email address of the service account. If this service account is not specified, the container runs as a service account that doesn't have access to the resource project.  
+              <br>
+              Users deploying the Model must have the iam.serviceAccounts.actAs permission on this service account.
+          - !ruby/object:Api::Type::Boolean
+            name: 'enableContainerLogging'
+            description: |
+              If true, the container of the DeployedModel instances will send stderr and stdout streams to Stackdriver Logging.  
+              Only supported for custom-trained Models and AutoML Tabular Models.
+          - !ruby/object:Api::Type::String
+            name: 'enableAccessLogging'
+            description: |
+              These logs are like standard server access logs, containing information like timestamp and latency for each prediction request.  
+              Note that Stackdriver logs may incur a cost, especially if your project receives prediction requests at a high queries per second rate (QPS). Estimate your costs before enabling this option.
+          - !ruby/object:Api::Type::NestedObject
+            name: 'privateEndpoints'
+            input: false
```

Comment:
```suggestion
```

Same as above- `input: false` is assumed, unless the resource is `input: true` (the same is true for other occurences)